definitions/features/only_foreman.rb

```diff
@@ -0,0 +1,9 @@
+class Features::OnlyForeman < ForemanMaintain::Feature
+  metadata do
+    label :only_foreman
```

Comment:
A couple of ideas for this and the Katello one as well:

 * foreman_scenario / katello_scenario (ties it to the installation type)
 * foreman_installer_scenario / katello_installer_scenario (same as above but more verbose)
 * foreman_install / katello_install